### ISSUE TITLE
List the duplicated images if dry-run is set to True

### ISSUE THREAD

**Issue by indam** [simulot/immich-go#13]
Thanks for your useful tool. I think it is better for users to review the duplicated files before deleting them, or let people know , even choose, which files will be deleted. 

Thank you.

**Comment by simulot** [comment]
> Thanks for your useful tool.

I'm pleased to know this tool is useful, thanks!

Granted the default output is useless.

It's quite possible to see details of the actions by changing the log level to INFO. Its looks like a debug log.

```sh
./immich-go -log-level=INFO  -server=http://your-server2283 -key=dU3YPigoB....3bU duplicate
Server status: OK
Connected, user: xxxxxx
Get server's assets...280 received
280 medias, 74 duplicate(s)...280 medias, 74 duplicate(s). Analyze completed.
2018-09-30T13:20:53Z:
   IMG_20180930_152053(5d361800-9ac6-4119-91a0-5760ca4bb396) 3648x2736, 963375 bytes: delete
   IMG_20180930_152053(4481db76-9844-488e-949f-09e1bf5ec7ae) 3648x2736, 2885802 bytes: keep
Skip deleting following 5d361800-9ac6-4119-91a0-5760ca4bb396, dry run mode
Skip Adding 4481db76-9844-488e-949f-09e1bf5ec7ae to album 80d7db7c-8237-4c3f-a77c-75b352b7c381, dry run mode
2018-09-30T13:20:55Z:
   IMG_20180930_152055(6681d84b-f67c-4e71-a978-d83958828f2b) 3648x2736, 816488 bytes: delete
   IMG_20180930_152055(05e5972d-e899-48c5-b4e5-cca68de4ca5e) 3648x2736, 2819303 bytes: keep
Skip deleting following 6681d84b-f67c-4e71-a978-d83958828f2b, dry run mode
Skip Adding 05e5972d-e899-48c5-b4e5-cca68de4ca5e to album 80d7db7c-8237-4c3f-a77c-75b352b7c381, dry run mode
....
```


Definitively, this command deserve a better output.

**Comment by simulot** [comment]
This feature is merged to main branch and will be added to the next release.

Here is the output:

``` sh
Server status: OK
Get server's assets...275 received
275 medias, 69 duplicate(s). Analyze completed.
There are 2 copies of the asset IMG_20180930_154721, taken on 2018-09-30T13:47:21Z 
  IMG_20180930_154721 3648x2736, 416.7 KB, upload/library/admin/2018/2018-09-30/IMG_20180930_154721.jpg to be deleted
  IMG_20180930_154721 3648x2736, 2.3 MB, upload/library/admin/2018/2018-09-30/IMG_20180930_154721+1.jpg to be kept
Proceed? [n]/y: n
There are 2 copies of the asset IMG_20180930_154729, taken on 2018-09-30T13:47:29Z 
  IMG_20180930_154729 3648x2736, 483.5 KB, upload/library/admin/2018/2018-09-30/IMG_20180930_154729.jpg to be deleted
  IMG_20180930_154729 3648x2736, 2.4 MB, upload/library/admin/2018/2018-09-30/IMG_20180930_154729+1.jpg to be kept
Proceed? [n]/y: ^C
Ctrl+C received. Shutting down...
```

**Comment by indam** [comment]
Thank you for the super fast response, looking forward to using it.

**Comment by simulot** [comment]
Included in v0.2.4
